0000-template.md

```diff
@@ -1,102 +1,125 @@
 ---
-feature name: (fill me in with a unique identifier, my-awesome-feature)
-start date: (fill me in with today's date, YYYY-MM-DD)
-rfc pr: (leave this empty)
-related issue: (tracking issue number)
+rfc pr: [#xxx](https://github.com/aws/aws-cdk-rfcs/pull/xxx) <-- fill this after you've already created the PR
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/yyy
 ---
 
-# Summary
+# [TITLE]
 
-> Brief description of the feature.
+> One sentence: brief description of the feature from a user perspective.
 
-# README
+## Working Backwards
```

Comment:
Besides these, we could consider an 'AWS What's New' post and blog post as good working backwards docs.

If you think so too, maybe mention it somewhere.

Reply:
What's new is the same as CHANGELOG, and a blog post is the same as README. At the moment I think those are sufficient.